gRPC/src/main/proto/promotion.proto

```
@@ -0,0 +1,37 @@
+syntax = "proto3";
+
+package com.sub.grpc;
+
+
+import "google/protobuf/empty.proto";
+import "google/protobuf/timestamp.proto";
+import "cardCompany.proto";
```

Buf (1.58.0)

import "cardCompany.proto": file does not exist

(COMPILE)